src/main/k/working/tests/overloads/domain-values/prove/domain-values-spec.k

```diff
@@ -0,0 +1,9 @@
+module VERIFICATION
+  imports DOMAIN-VALUES
+endmodule
+
+module DOMAIN-VALUES-SPEC
+  imports VERIFICATION
+  rule <k> 1 => end </k>
+  rule <k> (ctor a) => end </k>
```

Comment:
are you sure here `ctor a` is parsed as `Sub` and not directly as `Sup`?

Reply:
Better now?